tests/build/common_include_dir.srctree

```diff
@@ -21,13 +21,9 @@ import os
 
 from distutils.core import setup
 
-# os x on Travis specifically seems to crash with nthreads>0
-osx_on_travis = (platform.system() == "Darwin" and os.getenv("TRAVIS"))
-
 # Test concurrent safety if multiprocessing is available.
-# (In particular, TravisCI does not support spawning processes from tests.)
 nthreads = 0
-if not (hasattr(sys, 'pypy_version_info') or osx_on_travis):
+if not hasattr(sys, 'pypy_version_info'):
```

Comment:
I note that you've disable some of the OSX tests as "running indefinitely". It's possible that something similar to this special case is needed for github actions? Maybe?

Reply:
Oh, true, will revert this and add more. Im currently testing the pipelines against #3910 as all the pipelines seem to have succeeded there.

Reply:
Checked this gain, and now they both take longer to run due to not being threaded (2.7 and 3.8) with 3.8 still running indefinitely